reactor-core/src/main/java/reactor/util/Loggers.java

```diff
@@ -423,7 +423,8 @@ private String format(@Nullable String from, @Nullable Object... arguments){
 				String computed = from;
 				if (arguments != null && arguments.length != 0) {
 					for (Object argument : arguments) {
-						computed = computed.replaceFirst("\\{\\}", Matcher.quoteReplacement(argument.toString()));
+						computed = computed.replaceFirst("\\{\\}", Matcher.quoteReplacement(
+								argument != null ? argument.toString() : "null"));
```

Comment:
the whole can be replaced by `computed = computed.replaceFirst("\\{\\}", Matcher.quoteReplacement(String.valueOf(argument)));``